### ISSUE TITLE
A Bug? [glass.py: line 271]

### ISSUE THREAD

**Issue by SimpleXP** [cqylunlun/GLASS#4]
should `self.c /= len(training_data)` in upper level with the for loop?

**Comment by SimpleXP** [comment]
this part:

    for i, data in enumerate(training_data):
        img = data["image"]
        img = img.to(torch.float).to(self.device)
        batch_mean = torch.mean(img, dim=0)
        if i == 0:
            self.c = batch_mean
        else:
            self.c += batch_mean
        self.c /= len(training_data)

should be:

    for i, data in enumerate(training_data):
        img = data["image"]
        img = img.to(torch.float).to(self.device)
        batch_mean = torch.mean(img, dim=0)
        if i == 0:
            self.c = batch_mean
        else:
            self.c += batch_mean
    self.c /= len(training_data)

right?

**Comment by cqylunlun** [comment]
Thank you very much for your careful review. You clearly pointed out the bug. We have found that it was introduced when we accidentally indented `self.c /= len(training_data)` while removing Chinese comments. The bug at [line 271](https://github.com/cqylunlun/GLASS/blob/6e5218df4d49af46dfefb67f37d64fb3a8843a47/glass.py#L271) in glass.py has now been fixed.
